### PR TITLE
[BACKLOG-36641] Update and validate in-product links in 9.4

### PR DESCRIPTION
**Pull request by ddiroma**

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 5m 6s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **73**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/common-ui-11/job/PR-1594/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: docs/platform/visual/analyzer-viz-api.md [hunk 1]
```diff
@@ -23,8 +23,8 @@ while still side-by-side with the _previous_ Visualization API version.
 The Analyzer application supports visualizations of both formats,
 allowing you to convert any custom visualizations of the previous format at your own pace.
 
-All [stock visualizations](https://help.pentaho.com/Documentation/8.3/Products/Visualizations_for_Analyzer), 
-with the exception of the [Geo Map](https://help.pentaho.com/Documentation/8.3/Products/Visualizations_for_Analyzer#Geo_map_visualization), 
+All [stock visualizations](https://help.pentaho.com/Documentation/9.4/Products/Visualizations_for_Analyzer), 
+with the exception of the [Geo Map](https://help.pentaho.com/Documentation/9.4/Products/Visualizations_for_Analyzer#Geo_map_visualization),
```
Comment (dcleao) [inline docs/platform/visual/analyzer-viz-api.md:27]: The correct base URL is https://help.hitachivantara.com/Documentation/Pentaho/9.4/Products/Visualizations_for_Analyzer

Comment (rmansoor) [reply docs/platform/visual/analyzer-viz-api.md:27]: https://github.com/pentaho/pentaho-platform-plugin-common-ui/pull/1597

**Comment by rmansoor** [comment]
I have pushed the change using a different PR

**Pull request closed without merging**